src/AssetsTransferApi.ts

```diff
@@ -278,8 +294,13 @@ export class AssetsTransferApi {
 			assetIds,
 			amounts,
 			xcmDirection,
+			xcmPallet,
 			_specName,
 			registry,
+			{
```

Comment:
If this is an option should it be at the end of the args? And I think`xcmVersion`, and `paysWithFeeDest` can be added to it.

Reply:
good catch. adding those to the options